src/platform/forms-system/src/js/fields/ObjectField.jsx

```diff
@@ -255,68 +248,6 @@ class ObjectField extends React.Component {
         : idSchema.$id
     }${typeof pageIndex === 'undefined' ? '' : `_${pageIndex}`}`;
 
-    const fieldContent = (
-      <div className={containerClassNames}>
```

Comment:
We no longer need this since `accessibleFieldContent` has the proper markup structure.